src/components/controls/AlertNotice.svelte

```diff
@@ -0,0 +1,73 @@
+<script>
+  import { fade } from 'svelte/transition';
+  import { cubicOut } from 'svelte/easing';
+
+  // So we don't pollute the entire LS namespace.
+  const KEY_PREFIX = 'alertNotice-';
+  export let dismissText = 'dismiss'; // Label of 'dismiss' button.
+
+  // Required: Which localStorage key to check whether this notice has been dismissed.
+  export let toggleKey = '';
+
+  $: alertVisible = window.localStorage.getItem(KEY_PREFIX + toggleKey) !== 'true';
+
+  function dismissNotice() {
+    alertVisible = false;
+    window.localStorage.setItem(KEY_PREFIX + toggleKey, 'true');
+  }
+</script>
+
+<style>
+  .alert-notice {
+    position: absolute;
```

Comment:
Maybe `position: sticky` instead, so that it remains at the bottom-left when the page scrolls?

Reply:
Seems a little intrusive but I'll see what the consensus is. Easy to change.